build/local_builder.go

```diff
@@ -285,9 +367,9 @@ func (l *LocalBuilder) calculateLayerHashAndPlaceIt(layer *v1.Layer, tempTarget
 
 func (l *LocalBuilder) UpdateImageMetadata() error {
 	l.setClusterFileToImage()
-	if err := l.squashBaseImageLayerIntoCurrentImage(); err != nil {
```

Comment:
Canceling squash will make the current image start from "from layer", do you consider the consequence?